File: B/PHP/Ram.php
<?php

// Kelas Ram
class Ram
{
    private $capacity;
    private $price;

    // Constructor
    function __construct($capacity, $price) 
    {
        $this->capacity = $capacity;
        $this->price = $price;
    }

    // Metode Setter
    function setCapacity($capacity) 
    {
        $this->capacity = $capacity;
    }

    function setPrice($price) 
    {
        $this->price = $price;
    }

    // Metode Getter
    function getCapacity() 
    {
        return $this->capacity;
    }

    function getPrice() 
    {
        return $this->price;
    }

    // Metode Print
    function printRam() 
    {
        echo "===============================". "<br/>";
        echo "Ram:". "<br/>";
        echo "===============================". "<br/>";
        echo "Kapasitas     : ". $this->getCapacity(). "<br/>";
        echo "Harga         : ". "Rp". $this->getPrice(). "<br/>";
    }
}
?>

File: A/C++/Ram.cpp
#include <string>
#include <iostream>

using namespace std;

// Kelas RAM
class Ram
{
    // Atribut Private
    private:
        string capacity;
        int price;

    // Atribut Public
    public:
        // Constructor
        Ram()
        {
        
        }

        // Constructor Input
        Ram(string capacity, int price)
        {
            this->capacity = capacity;     
            this->price = price;
        }

        // Metode Setter
        void setCapacity(string capacity)
        {
            this->capacity = capacity;
        }

        void setPrice(int price)
        {
            this->price = price;
        }

        // Metode Getter
        string getCapacity()
        {
            return this->capacity;
        }

        int getPrice()
        {
            return this->price;
        }

        // Metode Print
        void printRam()
        {
            cout << "===============================" << endl;
            cout << "RAM:" << endl;
            cout << "===============================" << endl;
            cout << "Kapasitas      : " << this->capacity << endl;
            cout << "Harga          : " << "Rp" << this->price << endl;
        }
        // Destructor
        ~Ram()
        {

        };
};